src/main/java/Botbot.java

```diff
@@ -0,0 +1,166 @@
+//import scanner
+import java.util.Scanner;
+
+public class Botbot {
+    public static String line = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+
+    //create array for list
+    public static Task[] list = new Task[100];
+    public static int listSize = 0;
+
+    //method to mark or unmark task
+    public static void markUnmarkTask(String command){
```

Comment:
Noticed missing space between ) and {
```suggestion
    public static void markUnmarkTask(String command) {
```